[TOC]
## try-catch
 - 场景
 -  - 外部函数调用
 -  - 合约创建
 -  - 低级调用
 - 注意
 -  - try-catch  returns不分是可选的；但是如果出现就需要保持一致
 - try内部
 -  - 可以做很多事情
 - catch类型
 -  - catch Error(string memory reason){}
 -  - panic
 -  - bytes
 -  - 也可以不指定接受类型，直接就是{}
 - 自身代码块内的error
 -  - 会导致回滚
 - 被调用者
 -  - 因为发生了error 是肯定会回退的
 - 最多传递63/64
 -  - 防止无法处理错误